### PR TITLE
Backend implementation of the service list view

### PR DESCRIPTION
**Pull request by bryk**
1. This is a model for mocks:
https://github.com/kubernetes/dashboard/blob/master/designs/11-11-2015/16_card%20variations.png
2. Did little renaming (used the word "microservice") until we agree what the list exactly shows.
3. The implementation is very basic and will evolve over time.

### CODE REVIEW

File: src/app/backend/microservicelist.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	"k8s.io/kubernetes/pkg/labels"
+)
+
+// List of microservices in the cluster.
+type MicroserviceList struct {
+	// Unordered list of microservices.
+	Microservices []Microservice `json:"microservices"`
+}
+
+// Microservice is a replica set plus zero or more services.
```
Comment (PeWu) [inline src/app/backend/microservicelist.go]: Where are the services?

Comment (bryk) [inline src/app/backend/microservicelist.go]: Oh. I meant "Kubernetes services". Should I clarify?

Comment (bryk) [inline src/app/backend/microservicelist.go]: Clarified.

File: src/app/backend/microservicelist.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+type Microservice struct {
+	// Name of the microservice, derived from the replica set.
+	Name string `json:"name"`
+
+	// Replica set that represents the microservice.
+	ReplicaSet ReplicaSet `json:"replicaSet"`
+}
```
Comment (bryk) [inline src/app/backend/microservicelist.go]: Added todo.

**Comment by PeWu** [comment]
LGTM

**Pull request merged**